src/deckgl-layers/h3-hexagon-cell-layer/h3-hexagon-cell-layer.js

```diff
@@ -20,13 +20,40 @@
 
 import {HexagonCellLayer} from 'deck.gl';
 import {CylinderGeometry} from 'luma.gl';
-import {getAngle, getRadius, getH3VerticeTransform, transformCylinderPositions} from './h3-utils';
+import {getAngle, getRadius, getH3VerticeTransform, distortCylinderPositions} from 'layers/h3-hexagon-layer/h3-utils';
+import {editShader} from 'deckgl-layers/layer-utils/shader-utils';
+
+function addInstanceCoverage(vs) {
+  const addDecl = editShader(
```

Comment:
Unrelated to this diff, but noticed a typo in `editShader`, param is named `testToReplace` instead of `textToReplace`